docs/src/main/asciidoc/grpc-getting-started.adoc

```diff
@@ -64,7 +64,7 @@ Alternatively to using the `generate-code` goal of the `quarkus-maven-plugin`, y
 Let's start with a simple _Hello_ service.
 Create the `src/main/proto/helloworld.proto` file with the following content:
 
-[source]
+[source,javascript]
```

Comment:
why javascript?

Reply:
maybe `[source,protobuf]` will work?

Reply:
`protobuf` is not recognized, plain text is rendered. That's why I used `javascript` keyword to get some highlighting.

Reply:
oh, I see. 

I checked the available languages and SASS looks interesting:
![image](https://user-images.githubusercontent.com/3901322/110542828-d90bbf00-8129-11eb-9430-19ff8f8ea821.png)